core/tee/tee_svc_storage.c

```diff
@@ -657,6 +657,56 @@ TEE_Result tee_svc_storage_obj_del(uint32_t obj)
 	return TEE_SUCCESS;
 }
 
+TEE_Result tee_svc_storage_obj_del1(uint32_t obj)
```

Comment:
We're likely to be stuck with the legacy functions for a long time, better make it maintainable during that time.